#!/usr/bin/env python3

PACKAGE = "argus_computer_vision"

from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator()

gen.add("Width_Top", int_t, 0, "Width Top", 60, 0, 280)
gen.add("Height_Top", int_t, 0, "Height Top", 40, 0, 360)
gen.add("Width_Bottom", int_t, 0, "Width Bottom", 0, 0, 280)
gen.add("Height_Bottom", int_t, 0, "Height Bottom", 360, 0, 360)

gen.add("THRES", int_t, 0, "THRES", 60, 10, 150)
gen.add("MIN_AREA", int_t, 0, "MIN_AREA", 10000, 200, 50000)
gen.add("MAX_AREA", int_t, 0, "MAX_AREA", 18000, 200, 50000)

exit(gen.generate(PACKAGE, "argus_computer_vision", "lineParam"))
